### PR TITLE
FLPATH-340: propertyPath in update parameter response

### PR DESCRIPTION
**Pull request by RichardW98**
**What this PR does / why we need it**:
user needs to return propertyPath for dynamic parameters update.  Validation on  the parameters and propertyPath  is also needed for UI

**Which issue(s) this PR fixes** :
Fixes #[FLPATH-340](https://issues.redhat.com//browse/FLPATH-340)

**Change type**
- [ ] New feature
- [x] Bug fix
- [x] Unit tests
- [ ] Integration tests
- [ ] CI
- [ ] Documentation
- [ ] Auto generated SDK code

**Impacted services**
- [x] Workflow Service
- [ ] Notifivcation Service

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

File: workflow-service/src/main/java/com/redhat/parodos/workflow/definition/parameter/WorkParameterServiceImpl.java [hunk 1]
```diff
@@ -33,10 +39,30 @@ public List<WorkParameterValueResponseDTO> getValues(String workflowDefinitionNa
...
+
+		String propertyPath = workName;
+		WorkFlowDefinition workFlowDefinition = workFlowDefinitionService.getParentWorkFlowByWorkName(workName);
+
+		while (workFlowDefinition != null) {
+			propertyPath = String.join(".", workFlowDefinition.getName(), propertyPath);
+			workFlowDefinition = workFlowDefinitionService.getParentWorkFlowByWorkName(workFlowDefinition.getName());
+		}
+
+		return WorkParameterValueResponseDTO.builder().key(workParameterValueResponse.getKey())
```
Comment (eloycoto) [inline workflow-service/src/main/java/com/redhat/parodos/workflow/definition/parameter/WorkParameterServiceImpl.java:63]: could we move this to a static method inside the DTO object?

Comment (RichardW98) [reply workflow-service/src/main/java/com/redhat/parodos/workflow/definition/parameter/WorkParameterServiceImpl.java:63]: yes

**Review by pkliczewski — COMMENTED** [review]
/lgtm

**Review by anludke — APPROVED** [review]
/lgtm

**Pull request merged**